src/react-app-env.d.ts

```diff
@@ -1 +1,18 @@
 /// <reference types="react-scripts" />
+interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+}
+
+interface User {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+}
+
+interface TodoList extends Todo {
```

Comment:
TodoList interface extends Todo interface, but it should not. Consider changing `extends` to `implements` or removing it altogether. 🤔